Libraries/LibWeb/IndexedDB/Internal/Index.h

```diff
@@ -36,7 +36,7 @@ class Index : public JS::Cell {
     [[nodiscard]] bool unique() const { return m_unique; }
     [[nodiscard]] bool multi_entry() const { return m_multi_entry; }
     [[nodiscard]] GC::Ref<ObjectStore> object_store() const { return m_object_store; }
-    [[nodiscard]] AK::ReadonlySpan<IndexRecord> records() const { return m_records; }
+    [[nodiscard]] AK::Vector<IndexRecord> records() const { return m_records; }
```

Comment:
Having this return ReadonlySpan was kinda nice, do we need to change to Vector?

Reply:
It should work fine with `ReadonlySpan`, it was just missing the `first_matching` and `last_matching` methods 😅